src/components/buttons.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+
```

Comment:
De bestandsnamen van de componenten(Button,Product,Tile) moeten met een hoofdletter beginnen, de mapnamen niet.

Het Buttons component bevat één button zoals de bedoeling is, het is duidelijker om de naam dan enkelvoud te maken.